src/core/AutoRest.Core/Validation/Rule.cs

```diff
@@ -19,66 +16,23 @@ protected Rule()
         }
 
         /// <summary>
-        /// The name of the exception that describes this rule
+        /// The template message for this Rule. 
```

Comment:
I moved the connection of the `Message` and the  `Severity` to the Rule, so we didn't have to maintain a dictionary elsewhere.